app/controllers/container_controller.rb

```diff
@@ -204,7 +204,7 @@ def get_node_info(treenodeid)
     @sb[:action] = nil
     if x_node == "root" || TreeBuilder.get_model_for_prefix(@nodetype) == "MiqSearch"
       typ = "Container"
-      process_show_list
+      process_show_list(:where_clause => 'container_groups.deleted_on IS NULL')
```

Comment:
This pr depends upon #6369 see [comment](https://github.com/ManageIQ/manageiq/pull/6658#issuecomment-183888470)